runtimes-common/ref_runtimes_metering_labels.adoc

```diff
@@ -7,6 +7,11 @@
 
 You can add metering labels to your {ProductName} pods and check Red Hat subscription details with the OpenShift Metering Operator.
 
+[NOTE]
+====
+The OpenShift Metering Operator is not available in OpenShift version 4.9 and later.
+====
```

Comment:
I don't know @inoxx03 This seems like we're documenting OCP a bit too much. What is the reader supposed to do with this information? Is there an alternative? Should we advise them of any action to take?

Perhaps we should hold off until it becomes clear what the next step is going to be.

Reply:
@oraNod @inoxx03 There are QE from other teams who do not want to add Metering information in their guides because the "Additional resources" section links to OCP Metering guide of 4.8, where they have added deprecation note. This information should be available in our guides as well.
There is no Metering guide available from 4.9, because the operator has been removed. So when Runtimes start supporting 4.9 (which is a bleak probability) our guides need to mention that from 4.9 there is no metering operator.
As a mentioned in the mail, we do not have a replacement right now and might not have it for some time in the future. As mentioned in my mail, I spoke to John and the team who is working on the next operator. Until then we have to tell the users there is no Metering available if they are planning to move to OCP 4.9. Till 4.8 they have support for the Metering operator. So technically no user action is required because there is no operator right now. This note is FYI for user about metering operator. 
Hope this helps, or we can discuss on IM too (Stefan, you, and me).

Reply:
Also, @inoxx03 @oraNod putting same comment as our mail discussion:

I checked with John yesterday. He said this label may not be so frequently used by users. 
Would adding the product version as ":product-version: YYYY-QX" where QX is the quarter in which the product was released be more helpful to the user instead of Q1, because then the user will think Q1 is the default value.

Existing label description:
:product-version: YYYY-Q1 

Thinking aloud here. wdyt?

Reply:
@sangeetaraghu I think that using `YYYY-QX` as the product version placeholder is a good idea. No one has pointed the issue with `Q1` being confusing so far, but it is a possibility. I can change it in this PR.

Reply:
@oraNod IMHO one possible case when this note might be useful is for product releases that are supported on multiple OpenShift versions, one of which is 4.9 or later. The note tells the user that adding the labels will be no use on OCP on the later version, while it is still OK to use them for apps that they deploy to a cluster that runs on the earlier version.

Reply:
@inoxx03 I feel this needs to be more specific. How about something like:

As of OpenShift Container Platform version 4.9, the OpenShift Metering Operator is no longer available. There is currently no direct replacement for adding metering labels to pods. However you can continue using metering labels on OpenShift version 4.8 or earlier.

Reply:
Also we should probably combine this with the other note about not applying labels to pods that an operator manages.

Reply:
`YYYY-QX` is OK. I'm not a huge fan of this because it's not that meaningful and imo it's easier to understand something like 2021-Q4. If I were applying the metering labels I'd most likely see that and update it to the current value.